Unity/Assets/MyScripts/MyAppStart.cs

```diff
@@ -3,93 +3,91 @@
+    public static string uniqueUserId = "demoUserUnity";
 
-
-	// Use this for initialization
-	void Start ()
-	{
-		#if UNITY_ANDROID
+    // Use this for initialization
+    void Start()
+    {
+#if UNITY_ANDROID
         string appKey = "85460dcd";
-		#elif UNITY_IPHONE
+#elif UNITY_IPHONE
         string appKey = "8545d445";
-		#else
+#else
         string appKey = "unexpected_platform";
-		#endif
-		Debug.Log ("unity-script: MyAppStart Start called");
-
-		//Dynamic config example
-		IronSourceConfig.Instance.setClientSideCallbacks (true);
-
-		string id = IronSource.Agent.getAdvertiserId ();
-		Debug.Log ("unity-script: IronSource.Agent.getAdvertiserId : " + id);
-		
-		Debug.Log ("unity-script: IronSource.Agent.validateIntegration");
-		IronSource.Agent.validateIntegration ();
-
-		Debug.Log ("unity-script: unity version" + IronSource.unityVersion ());
-
-		// Add Banner Events
-		IronSourceEvents.onBannerAdLoadedEvent += BannerAdLoadedEvent;
-		IronSourceEvents.onBannerAdLoadFailedEvent += BannerAdLoadFailedEvent;		
-		IronSourceEvents.onBannerAdClickedEvent += BannerAdClickedEvent; 
-		IronSourceEvents.onBannerAdScreenPresentedEvent += BannerAdScreenPresentedEvent; 
-		IronSourceEvents.onBannerAdScreenDismissedEvent += BannerAdScreenDismissedEvent;
-		IronSourceEvents.onBannerAdLeftApplicationEvent += BannerAdLeftApplicationEvent;
-
-		// SDK init
-		Debug.Log ("unity-script: IronSource.Agent.init");
-		IronSource.Agent.init (appKey);
-		//IronSource.Agent.init (appKey, IronSourceAdUnits.REWARDED_VIDEO, IronSourceAdUnits.INTERSTITIAL, IronSourceAdUnits.OFFERWALL, IronSourceAdUnits.BANNER);
+#endif
+        Debug.Log("unity-script: MyAppStart Start called");
+
+        //Dynamic config example
+        IronSourceConfig.Instance.setClientSideCallbacks(true);
+
+        string id = IronSource.Agent.getAdvertiserId();
+        Debug.Log("unity-script: IronSource.Agent.getAdvertiserId : " + id);
+
+        Debug.Log("unity-script: IronSource.Agent.validateIntegration");
+        IronSource.Agent.validateIntegration();
+
+        Debug.Log("unity-script: unity version" + IronSource.unityVersion());
+
+        //Add AdInfo Banner Events
+        IronSourceBannerEvents.onAdLoadedEvent += BannerOnAdLoadedEvent;
+        IronSourceBannerEvents.onAdLoadFailedEvent += BannerOnAdLoadFailedEvent;
+        IronSourceBannerEvents.onAdClickedEvent += BannerOnAdClickedEvent;
+        IronSourceBannerEvents.onAdScreenPresentedEvent += BannerOnAdScreenPresentedEvent;
+        IronSourceBannerEvents.onAdScreenDismissedEvent += BannerOnAdScreenDismissedEvent;
+        IronSourceBannerEvents.onAdLeftApplicationEvent += BannerOnAdLeftApplicationEvent;
+
+        // SDK init
+        Debug.Log("unity-script: IronSource.Agent.init");
+        IronSource.Agent.init(appKey);
+        //IronSource.Agent.init (appKey, IronSourceAdUnits.REWARDED_VIDEO, IronSourceAdUnits.INTERSTITIAL, IronSourceAdUnits.OFFERWALL, IronSourceAdUnits.BANNER);
```

Comment:
I think we don't need this

Reply:
also the line below it can be removed although it's commented

Reply:
I was going to remove the line below, but I think it was added on purpose to show the different wait to Init the sdk

Reply:
Hi @virginiecrequit and @mahdikahia 
Why did you decide to remove the example of the init of a specific ad unit?
did it was defined by the product/ticket?

Reply:
was commented thats why I assumed it wasn't needed,

you can bring it back @virginiecrequit

Reply:
Done!